ci/before_install.sh

```diff
@@ -58,3 +58,7 @@ if [[ -n "$CMAKE_URL" ]]; then
     exit 1
   }
 fi
+
+echo "Install perl"
+curl -L https://install.perlbrew.pl | bash
+perlbrew install --notest perl-5.30.1
```

Comment:
Can you test on the perl version used in Travis? Update `.travis.yml` for apt packages if needed.
Do the same for Appveyor.  Use https://github.com/lukesampson/scoop to install perl and cpanm if Appeveyor has no convenient helpers for perl.

If your plugin must work on the latest perl version only, update your PR description and add a health check to require this latest perl version.

Reply:
The module does not require a specific version of perl, so any version is fine, I'll update the PR accordingly. As far as I know, supporting remote plugins on Windows is problematic given the issues with ```poll```-ing stdin? To date perl doesn't have an event loop implementation that can actually poll stdin (I've started working on a libuv-based implementation, but that is nowhere near being usable nor complete). I would expect the node provider to work, but I'm unaware of the state of python/ruby providers?

Reply:
https://travis-ci.org/neovim/neovim/jobs/629215538#L1

```
perlbrew version
/home/travis/perl5/perlbrew/bin/perlbrew  - App::perlbrew/0.86
```

Is that recent enough if you need to use perlbrew?